[BugFix] Hide "Visualize argument values" entry for "instance_id" arg…

KingAlen opened the pull request:
… key

Summary of the change:

The 'Focus LynxView' plugin introduces an 'instance_id' table, where every trace event with an ancestor containing the 'instance_id' arg key is also assigned this key and inserted into the table. This can result in a large number of trace events with the 'instance_id' arg key. Consequently, the "Visualize argument values" feature for the 'instance_id' arg key can become very costly due to extensive searches in the `ancestor_slice($id)` table. To optimize this, we now use the 'Focus LynxView' feature to replace "Visualize argument values" for this specific key, and have therefore hidden the "Visualize argument values" entry for the 'instance_id' arg key.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/lynx-family/lynx-trace?pullRequest=6) <br/>All committers have signed the CLA.

KingAlen committed:
[BugFix] Hide "Visualize argument values" entry for "instance_id" arg key

Summary of the change:

The 'Focus LynxView' plugin introduces an 'instance_id' table, where every trace event with an ancestor containing the 'instance_id' arg key is also assigned this key and inserted into the table. This can result in a large number of trace events with the 'instance_id' arg key.
Consequently, the "Visualize argument values" feature for the 'instance_id' arg key can become very costly due to extensive searches in the `ancestor_slice($id)` table. To optimize this, we now use the 'Focus LynxView' feature to replace "Visualize argument values" for this specific key, and have therefore hidden the "Visualize argument values" entry for the 'instance_id' arg key.
